### ISSUE TITLE
Findings for VSA a11y spot check review for "Modify dependents"

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#8258]
### For reference

**VSA Development Review** - [https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review)

## Team Name

vsa-ebenefits

## Product/Feature Name

"Modify dependents (using the 686 form)"

## Product description

The first part of a larger form application where a Veteran can first view what dependents are on or not on award for different benefits so they can take action and apply or notify for a change in benefit or compensation.

## Review stage

- [ ] **Development intent review** (Review design to discuss approach to take with development, identify what components and semantic HTML would be appropriate, identify any accessibility issues and how to address) - This would happen as designs are passed to development, before development begins.

- [x] **Spot check review** (Review and follow the steps in the [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md) first.) This happens well before launch, so that the 508 office can review and any issues addressed *before* launch.

- [ ] **Other:** Please indicate what you're requesting

## Point of contact

Jason Wolf (@jason-gcio) - PM
Micah Chiang (@micahchiang) - FE
Jesse Cohn (@cohnjesse) - FE

## Preferred times for dev intent checkpoint

Any time in the next sprint or 2, afternoons to accommodate west coasters 

## Staging URL and credentials

URL: ~~https://staging.va.gov/disability/view-dependents/~~
New url as of May 15, 2020: https://staging.va.gov/view-change-dependents/view/

  - Test-accounts `(usual staging password)`  
    - User (Mark): vets.gov.user+228@gmail.com  
    - User (Eric): vets.gov.user+10@gmail.com  
    - User (Jeanette): vets.gov.user+123@gmail.com  
    - User (Pauline): vets.gov.user+38@gmail.com  
    - User (Sam): vets.gov.user+45@gmail.com  
    - User (Claude): vets.gov.user+232@gmail.com  
    - User (Terrence): vets.gov.user+102@gmail.com  
    - User (Randall): vets.gov.user+99@gmail.com  
    - User (Jesse): vets.gov.user+54@gmail.com  
    - User (Jerry): vets.gov.user+271@gmail.com  

## Objectives:
Login as one or several different users
Ensure that all 508 compliance initiatives are met, including screen readers, magnification and mobile

## Precondition for Modify Dependents
Be logged in as an LOA 3 user: User (Jerry): vets.gov.user+271@gmail.com (has **2** on award, **4** not)

### A. Introduction
1. Browse to https://staging.va.gov/view-change-dependents/add-remove-form-686c/
- see the intro page
2. Click the button "Start the Application" (this button color and text may be adjusted)
- arrive at a screen seeking the selection of checkboxes.

### B. Checkboxes and Veteran Info
1.  Browse to https://staging.va.gov/view-change-dependents/add-remove-form-686c/686-options-selection
- see the 686 options page
2. Select "Claim additional benefits for a child
- a blue check box will be filled
3. Click blue Continue button 
- move to next screen which is seeking Veteran's Information (Name, SSN, DOB)
4. Complete required fields on form
- First and last name are required, SSN (796-14-8937) needs to have 9 digits, dashes allowed. A birthdate must be valid
5. Click blue Continue button 
- move to next screen which is seeking more Veteran's Information
6. Click first checkbox for military base
- Country becomes United States by default and cannot be changed.  "City" becomes "APO/FPO/DPO" both required
7. Click "Learn more about military base addresses"
- drop down informs you that "The United States is automatically chosen as your country if you live on a military base outside of the country."
8. Click "Learn more about military base addresses" again
- drop down rolls back up
9. Complete required fields on form
- required fields should be pretty wide open, zip code must be 5 digits. Only numbers for phone (8008271000) no dashes
10. Click blue Continue button
- move to next screen which is seeking Information meeded to add your child
11. Complete required fields on form
- required fields are all open text, child's SSN needs to be 9 digits but more are currently allowed (to be fixed).  When a letter is entered, it gives a technical description of what should be entered and will be fixed to be easier to read.
12. Add another child (my be updated to say "Next child" instead of "New Child")
- invited to fill out this small form, same issues as before
13. Click blue Continue button
- move to next screen which is seeking more information needed to add your child
14. Complete required fields on form
- required fields are required and the checkbox that indicates stepchild will dropdown to inquire about the required, valid, date when that happened
15. Any time a box other than biological is selected, additional evidence will be required. Click "What are some examples of additional evidence?"
- The helper text section will dropdown when clicked with some examples
16. Radio buttons asking if child was previously married
- Default is no, yes will open up a section asking for more information about those details, the required, valid, date when that marriage ended and the reason.  If "Other" reason is selected, a required area to write in a reason is made available.
17. Click blue Continue button
- move to next screen that continues to ask information about your child
18. You are asked if the child lives with you with radio buttons yes or no
- yes - nothing happens, no - enter the information to indicate where that child lives with required fields
19. Click blue Continue button
- move to next screen that requests additional evidence for non-biological children
20. Click the helper text that says "What are some examples of additional evidence?"
- The helper text section will dropdown when clicked with some examples
21. Upload a file like a jpg
- Uploading a .jpg resulted in an internal server error which I think might be expected.
22. Go back and select Biological as the child type to proceed.
- move to next screen to review the different sections
23. Expand 686c Options
- see the true statement near the selected wrokflow. (this should be updated to say something like "Selected" instead of "True")
24. the Privacy policy requirement is required to advance
- not checking that box shows that the action is required
25. Click "Submit application"
- The form tries to send the information in but fails with a nice red message "We’re sorry. We can't submit your form right now..."
26. Click the blue Go back to VA.gov
- Arrive at the home page
